csharp/extractor/Semmle.Extraction.CSharp/Entities/Keyword.cs

```diff
@@ -0,0 +1,20 @@
+internal static class Keyword
```

Comment:
The idea is to just extend the list of keywords in the file, when there is a need.
I will rename it to modifiers - then it can be renamed to keywords another time in case we start using it for more general purpose.
In any case, I would like to get rid of hardcoded strings when I encounter them in the extractor and have them at most one place - especially because many of them are used multiple times.